src/interpreter.c

```diff
@@ -424,7 +424,7 @@ void command_interpreter( struct char_data *ch, char *argument )
 
  /* PARRY is removed as HIDE GAIA 2001 */
 
-   if(IS_AFFECTED(ch,AFF2_PARRY))
+   if(IS_AFFECTED2(ch,AFF2_PARRY) && !ch->specials.fighting)
```

Comment:
Da quello che vedo qui è un fix solo cosmetico, ma su whatsappa avevi parllato anche di "Aggiunto un caso che previene il vecchio problema del retrigger in combat, impossibilitandone la disattivazione"